### ISSUE TITLE
Error on ListGroupMembersCount 

### ISSUE THREAD

**Issue by Yeoman742** [kylegregory/EmmaSharp#39]
Getting an exception when trying to get a count of members in a specific group.

Message = "Unexpected character encountered while parsing value: [. Path '', line 1, position 1."
StackTrace = "   at Newtonsoft.Json.JsonTextReader.ReadNumberValue(ReadType readType)\r\n   at Newtonsoft.Json.JsonTextReader.ReadAsInt32()\r\n   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ReadForType(JsonReader reader, JsonContract contract, Boolean ...
TargetSite = {System.Object ReadNumberValue(Newtonsoft.Json.ReadType)}

**Comment by kylegregory** [comment]
@Yeoman742 Do you expect there was a high number of members on the group? Higher than, say, an int32 would hold?

**Comment by Yeoman742** [comment]
No, we are only talking about anywhere between 6,000 to 20,000.  I think an int32 can old that fine.

**Comment by kylegregory** [comment]
@Yeoman742 May try updating to v1.3.0 just released. It updates a lot of the Emma "ids" to longs, and that may help. Otherwise, I'd need to know more about what is happening.
